# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1521765630 25200
#      Thu Mar 22 17:40:30 2018 -0700
# Branch stable
# Node ID c4a3b8b3fbd86dcf8c0e32e5c4ed7a2d65b2427c
# Parent  40e9c727ccfaddf7c7a1456a06aeecd6cd718cbf
rewrite desktop icon Comment entry to match AppStream summary

* org.octave.Octave.desktop.in: Rewrite Comment entry to match AppStream
summary tag and Octave homepage description.  Delete out of date translation
for now.

diff --git a/etc/icons/org.octave.Octave.desktop.in b/etc/icons/org.octave.Octave.desktop.in
--- a/etc/icons/org.octave.Octave.desktop.in
+++ b/etc/icons/org.octave.Octave.desktop.in
@@ -1,12 +1,11 @@
 [Desktop Entry]
 Name=GNU Octave
-Comment=Scientific Computing
-Comment[de]=Wissenschaftliches Rechnen
+Comment=Interactive programming environment for numerical computations
 TryExec=%OCTAVE_PREFIX%/bin/octave
 Exec=%OCTAVE_PREFIX%/bin/octave --gui %f
 Icon=octave
 Terminal=false
 Type=Application
 Categories=Education;Science;Math;
 StartupNotify=false
 StartupWMClass=octave-gui
